packages/components/src/internal/components/forms/input/RadioGroupInput.tsx

```diff
@@ -41,11 +41,12 @@ class RadioGroupInputImpl extends PureComponent<RadioGroupInputProps, State> {
     }
 
     onValueChange: ChangeEventHandler<HTMLInputElement> = (evt): void => {
-        const { formsy, onValueChange, setValue } = this.props;
-        const { value } = evt.target;
+        this.onSetValue(evt.target.value);
```

Comment:
*Note: Leaving a comment here because I can only comment on changed lines*

We should take this opportunity to convert this component to an FC, I also don't think it should return an array the way it is.

Reply:
Done, and changed so that it doesn't return an array anymore